packages/venia-ui/lib/components/CheckoutPage/OrderSummary/orderSummary.css

```diff
@@ -0,0 +1,17 @@
+.root {
+}
+
+.title {
+    padding-bottom: 2rem;
+}
+
+@media (max-width: 960px) {
+    .root {
+        border-top: 1px solid rgb(var(--venia-border));
+        padding-top: 2rem;
+    }
+
+    .title {
+        font-size: 1.25rem;
```

Comment:
According to a comment @jimbo made on [one of my other PRs](https://github.com/magento/pwa-studio/pull/2257#discussion_r396686556), font size should not be fractional and in pixels. `1.25rem` is `20px`. Can you change this?

Reply:
I didn't check that comment until now, I don't think that's what @jimbo meant (in context of @revanth0212's PR). You should still use `rem` units, but they should be whole pixel values. You were using `0.9 rem` which is `14.4 px`, no good.

@sirugh - I'm going to create a quick PR to revert this change.